test/runtask.test.js

```diff
@@ -13,48 +12,47 @@ test('setup', (t) => {
 test('runs basic fn', (t) => {
   t.plan(2);
   const runner = new RunTask();
-  runner.register('test', (done) => {
+  runner.register('test', (data, done) => {
     t.pass('fn is called');
     done();
   });
 
-  runner.run('test', () => {
+  runner.run('test', {}, () => {
```

Comment:
lets make the data optional on this step.  so inside run, it does the check and just sets `{}` as data